docs/book/src/installation/guest_physical/requirements.rst

```diff
@@ -59,3 +59,15 @@ However, any re-imaging platform can be used (Clonezilla, Deepfreeze, etc) to
 accomplish this.
 
 .. _`Fog`: http://www.fogproject.org/
+
+Cuckoo Configuration Requirements
+=================================
+Since we are using physical machines to perform our analysis, we must account
+for the reboot/rebuild time of our physical machines in our Cuckoo configuration.
+Spefically, we must modify the vm_state timeout as specified in conf/cuckoo.conf.
+	vm_state = 60
+By default, this value is set to 60 (seconds). We need to update it so that it
+reflects the amount of time required to reboot and rebuild the physical guest.
+In testing 10 minutes (i.e., vm_state = 600) has proven sufficient. However,
```

Comment:
Please add something like `with a normal Windows 7 setup and 1 gbit connection`.